crates/iota-core/src/consensus_adapter.rs

```diff
@@ -182,6 +182,87 @@ impl ConsensusAdapterMetrics {
     pub fn new_test() -> Self {
         Self::new(&Registry::default())
     }
+
+    pub fn unregister(&self, registry: &Registry) {
+        registry
+            .unregister(Box::new(self.sequencing_certificate_attempt.clone()))
+            .expect("Failed to unregister sequencing_certificate_attempt");
```

Comment:
We have quite a few new expect() panics, is this okay ?
(I suppose it won't actually stop the program as they're caught and recovered, but do we want to just print an error right here instead ?)

Reply:
If that code ever panics, there is a logic error somewhere and the developer made a mistake. IMO we should keep the panics, and add test cases that prevent that someone forgets to unregister a metrics.